bindings/wallet-c/wallet.h

```diff
@@ -110,6 +112,20 @@ typedef struct TimeEra
   uint32_t slots_per_epoch;
 } TimeEra;
 
+typedef struct SettingsInit
+{
+  struct LinearFee fees;
+  enum Discrimination discrimination;
+  /**
+   * block_0_initial_hash is assumed to point to 32 bytes of readable memory
+   */
+  const uint8_t *block0_initial_hash;
+  Block0Date block0_date;
```

Comment:
I don't know if we should have a type alias for this. There will be some confusion with the `BlockDate` struct, and this one is essentially a Unix timestamp.